playground/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/ai-lab-recipes/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
+	curl -s -S -L -f https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/resolve/main/llama-2-7b-chat.Q5_K_S.gguf -z $@ -o $@.tmp && mv -f $@.tmp $@ 2>/dev/null || rm -f $@.tmp $@
```

Comment:
Can we use this instead? https://huggingface.co/TheBloke/Mistral-7B-v0.1-GGUF/resolve/main/mistral-7b-v0.1.Q2_K.gguf 
Its smaller model (3gb) and has an apache license.

Reply:
That is doable. But, I would prefer implementing it on a followup PR.